include/vapor/DCBOV.h

```diff
@@ -0,0 +1,175 @@
+
+    //! \copydoc DC::GetAuxVarInfo()
+    //!
+    virtual bool getAuxVarInfo(string varname, DC::AuxVar &var) const { return (false); }
+
+    //! \copydoc DC::GetBaseVarInfo()
+    //
+    virtual bool getBaseVarInfo(string varname, DC::BaseVar &var) const;
+
+    //! \copydoc DC::GetDataVarNames()
+    //!
+    virtual std::vector<string> getDataVarNames() const;
+
+    virtual std::vector<string> getAuxVarNames() const { return (vector<string>()); }
+
+    //! \copydoc DC::GetCoordVarNames()
+    //!
+    virtual std::vector<string> getCoordVarNames() const;
+
+    //! \copydoc DC::GetCoordVarNames()
+    //!
+    virtual size_t getNumRefLevels(string varname) const { return (1); }
+
+    //! \copydoc DC::GetMapProjection()
+    //!
+    virtual string getMapProjection() const
+    {
+        return (_proj4String);
+    }
+
+    //! \copydoc DC::GetAtt()
+    //!
+    virtual bool getAtt(string varname, string attname, vector<double> &values) const;
+    virtual bool getAtt(string varname, string attname, vector<long> &values) const;
+    virtual bool getAtt(string varname, string attname, string &values) const;
+
+    //! \copydoc DC::GetAttNames()
+    //!
+    virtual std::vector<string> getAttNames(string varname) const;
+
+    //! \copydoc DC::GetAttType()
+    //!
+    virtual XType getAttType(string varname, string attname) const;
+
+    //! \copydoc DC::GetDimLensAtLevel()
+    //!
+    virtual int getDimLensAtLevel(string varname, int level, std::vector<size_t> &dims_at_level, std::vector<size_t> &bs_at_level) const;
+
+    //! \copydoc DC::OpenVariableRead()
+    //!
+    virtual int openVariableRead(size_t ts, string varname, int, int) { return (DCBOV::openVariableRead(ts, varname)); }
+
+    virtual int openVariableRead(size_t ts, string varname);
+
+    //! \copydoc DC::CloseVariable()
+    //!
+    virtual int closeVariable(int fd);
+
+    //! \copydoc DC::ReadRegion()
+    //
+    virtual int readRegion(int fd, const vector<size_t> &min, const vector<size_t> &max, float *region) { return (_readRegionTemplate(fd, min, max, region)); }
+    virtual int readRegion(int fd, const vector<size_t> &min, const vector<size_t> &max, int *region) { return (_readRegionTemplate(fd, min, max, region)); }
+
+    //! \copydoc DC::ReadRegionBlock()
+    //!
+    virtual int readRegionBlock(int fd, const vector<size_t> &min, const vector<size_t> &max, float *region) { return (_readRegionTemplate(fd, min, max, region)); };
+    virtual int readRegionBlock(int fd, const vector<size_t> &min, const vector<size_t> &max, int *region) { return (_readRegionTemplate(fd, min, max, region)); }
+
+    //! \copydoc DC::VariableExists()
+    //!
+    virtual bool variableExists(size_t ts, string varname, int reflevel = 0, int lod = 0) const;
+
+private:
+    VAPoR::UDUnits      _udunits;
+
+    BOVCollection *_bovCollection;
+
+    string                                      _varname;
+    string                                      _proj4String;
```

Comment:
BOV has no support for projection strings, correct? Suggest deleting this and just returning an empty string

Reply:
Fixed.